### PR TITLE
Update schedule to reflect more topics and dates

### PR DESCRIPTION
**Pull request by avahoffman**
- Add actual dates to config and schedule
- Add MVP topic to week 4
- Add a few more videos and other resources

### CODE REVIEW

**Comment by avahoffman** [comment]
@carriewright11 I think this is ready! I added the dates to the schedule and config since I know I struggled a bit with this last time. Let me know if you don't like the way it looks.

**Comment by carriewright11** [comment]
Hi! Noticed that we should change the time to EDT... or just ET?

File: schedule.Rmd [hunk 1]
```diff
@@ -17,21 +19,21 @@ Every class session will start and end with a writing prompt that will be used f
...
...me/coid/8136/)| * [Black Butterfly Academy: Baltimore Apartheid](https://jhubiostatistics.shinyapps.io/Baltimore_Apartheid/) <br> * [Mitigating Bias Training](https://jhudatascience.org/Baltimore_Community_Course/Assignments/Mitigating_Bias_Assignment.html) |**Reading About History:** <br> * [Greenwood Tulsa Massacre](https://www.nytimes.com/interactive/2021/05/24/us/tulsa-race-massacre.html)<br> * [Rosewood Massacre](https://en.wikipedia.org/wiki/Rosewood_massacre) <br> * [Red Summer](https://en.wikipedia.org/wiki/Red_Summer) <br> * [Not in My Neighborhood: How Bigotry Shaped a Great American City](https://www.anteropietila.com/not-in-my-neighborhood) <br> * [Henrietta Lacks: science must right a historical wrong](https://www.nature.com/articles/d41586-020-02494-z) <br> **More About Bias:** <br> * [Discrimination in Workplace Dynamics: Toward a Structural Account of Disparate Treatment Theory](http://web.archive.org/web/20120218231339/http://www.law.harvard.edu/students/orgs/crcl/vol38_1/green.pdf) <br> * [Take an Implicit Bias Test](https://implicit.harvard.edu/implicit/takeatest.html) <br> * [Bias in A.I. For Women & People of Color](https://www.youtube.com/watch?v=z1w0ZVrzZjw)|
```
Comment (carriewright11) [inline schedule.Rmd:32]: nice!

File: schedule.Rmd [hunk 2]
```diff
@@ -41,27 +43,27 @@ Every class session will start and end with a writing prompt that will be used f
...
+|This week we will discuss more specifically about how students will **interact** with partners, as well as examples of what has and has not worked well in past similar projects. Students will meet with their **project group** for the first time and start assigning **student roles**.|* Minimum Viable Products (MVPs) <br> * [Project planning form](https://docs.google.com/document/d/1b0XHuKsys_ACCtA9nI_M9F1CR2oWo2RLR-A1C6cvz0U/edit?usp=sharing) <br> * [Project organization and infrastructure suggestions](https://docs.google.com/document/d/1RveTbckBTomgx2JiiCbm01SE88OvODK7JCQxo1gbLbo/edit?usp=sharing)  |  * [Git & GitHub Assignment](https://jhudatascience.org/Baltimore_Community_Course/Assignments/Using_git_Assignment.html) |  * [SOURCE Dos and don'ts](https://source.jhu.edu/publications-and-resources/DosAndDontsOfPartnerships.pdf) <br> * [Project Student Roles](https://jhudatascience.org/Baltimore_Community_Course/projects.html) <br> * [Project Reminders](https://jhudatascience.org/Baltimore_Community_Course/projects.html)|
```
Comment (carriewright11) [inline schedule.Rmd:63]: is there an MVP link we could use? or maybe that will be our own slides?

Comment (avahoffman) [reply schedule.Rmd:63]: Yes, will add later. Just adding as a placehodler for now.

**Comment by avahoffman** [comment]
> Hi! Noticed that we should change the time to EDT... or just ET?

Thank you for fixing this! I never know which to use :P

**Pull request merged**